api/server/router/delta/delta_routes.go

```diff
@@ -0,0 +1,27 @@
+package delta
+
+import (
+	"net/http"
+
+	"github.com/docker/docker/api/server/httputils"
+	"github.com/docker/docker/api/types"
+	"golang.org/x/net/context"
+)
+
+func (d *deltaRouter) postDeltasCreate(ctx context.Context, w http.ResponseWriter, r *http.Request, vars map[string]string) error {
+	if err := httputils.ParseForm(r); err != nil {
+		return err
+	}
+
+	deltaSrc := r.Form.Get("src")
+	deltaDest := r.Form.Get("dest")
```

Comment:
This varnames kind of imply that these are deltas, when in reality these are images, right?